player/views/client/client_player.py

```diff
@@ -47,8 +48,30 @@ def next_music(request):
 def update_player(request):
     if request.is_ajax and request.session.get('room', False):
         room = Room.objects.get(name=request.session.get('room'))
-        player_updated = render_player(room=room)
-        return HttpResponse(player_updated, content_type='application/json')
+
+        musics = room.music_set.all().order_by('-date')
+        # Get the paginator
+        paginator = Paginator(musics, 16)
+        more_musics = False
+        try:
+            page = int(request.POST.get('page')) - 1
+        except ValueError:
+            page = 1
+            print("Error on page requested")
```

Comment:
send an error to the client or skip the request. We don't care about log bastards that send us shit :p